databricks/koalas/tests/test_series.py

```diff
@@ -15,13 +15,90 @@
 #
 
 
+import numpy as np
 import pandas as pd
 
 from databricks import koalas
+from databricks.koalas.series import Series
 from databricks.koalas.testing.utils import ReusedSQLTestCase, SQLTestUtils
 
 
 class DataFrameTest(ReusedSQLTestCase, SQLTestUtils):
+    @property
+    def pdf(self):
+        return pd.DataFrame({
+            'a': [1, 2, 3, 4, 5, 6, 7, 8, 9],
+            'b': [4, 5, 6, 3, 2, 1, 0, 0, 0],
+        }, index=[0, 1, 3, 5, 6, 8, 9, 9, 9])
+
+    @property
+    def kdf(self):
+        return koalas.from_pandas(self.pdf)
+
+    def test_Series(self):
```

Comment:
I think we should stick to the one naming `test_series` while we're here.

Reply:
Agree !